harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/localfilesystem/LocalFsHarvesterFileVisitor.java

```diff
@@ -112,17 +122,46 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
             if (file != null &&
                 file.getFileName() != null &&
                 file.getFileName().toString() != null &&
-                file.getFileName().toString().endsWith(".xml")) {
+                    (file.getFileName().toString().endsWith(".xml") || file.getFileName().toString().endsWith(".mef"))) {
```

Comment:
Note that current export to MEF (call `Export (ZIP)` in the UI) creates files with `zip` extension. 

Not really sure if should be checked that extension here. I think maybe would be better to change the export to MEF to use `mef` extension. But not something to change in this pull request.